spec/fixtures/slack/web/channels_info.yml

```diff
@@ -2,45 +2,138 @@
 http_interactions:
 - request:
     method: post
-    uri: https://slack.com/api/channels.list
+    uri: https://slack.com/api/conversations.list
     body:
       encoding: UTF-8
       string: token=token
+    headers:
+      Accept:
+      - application/json; charset=utf-8
+      User-Agent:
+      - Slack Ruby Client/0.15.0
+      Content-Type:
+      - application/x-www-form-urlencoded
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
```

Comment:
Do you think it's useful to leave all these additional headers in?

Reply:
I don't think it matters. I rarely edit anything in VCR.